apps/web/components/UtilityContent.tsx

```diff
@@ -16,9 +20,15 @@ import {
 export default function UtilityContent() {
   const { t } = useTranslation();
 
-  const overlayStyle = {
-    width: "250px",
-    borderRadius: "10px",
+  const [signUpModal, setSignUpModalVisibility] = useState<Boolean>(false);
+  const [signInModal, setSignInModalVisibility] = useState<Boolean>(false);
+
+  const handleSignUpModalVisibility = () => {
+    return !signUpModal ? setSignUpModalVisibility(true) : setSignUpModalVisibility(false);
+  };
+
+  const handleSignInModalVisibility = () => {
+    return !signInModal ? setSignInModalVisibility(true) : setSignInModalVisibility(false);
   };
```

Comment:
@MFelipeSilva instead of create a function to call another function,  you can pass the `open` props to the Modal component.